### PR TITLE
Dropdown component

### PR DESCRIPTION
**Pull request by kimdil89**
My first commit with dropdown component (with very basic styling, I will change it)

### CODE REVIEW

File: src/components/Dropdown/Dropdown.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    margin-top: -ceil(2.5);
+    position: absolute;
+    right: 10px;
+    top: 14px;
+    width: 0
+  }
+
+  .MyPlaceholderClassName {
+      width: 230px;
+  }
```
Comment (magul) [inline src/components/Dropdown/Dropdown.scss]: newline please

File: package.json [hunk 2]
```diff
@@ -25,9 +25,13 @@
...
     "path": "^0.12.7",
     "react": "16.8.6",
-    "react-dom": "16.8.6"
+    "react-dom": "16.8.6",
+    "react-dropdown": "^1.6.4",
+    "react-select": "^3.0.4"
   },
   "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
```
Comment (magul) [inline package.json]: How does `@babel/core` with `@babel/preset-env` and `babel-core` with`babel-preset-env` work together?

File: package.json [hunk 3]
```diff
@@ -25,9 +25,13 @@
     "nunjucks": "^3.2.0",
     "path": "^0.12.7",
     "react": "16.8.6",
-    "react-dom": "16.8.6"
+    "react-dom": "16.8.6",
+    "react-dropdown": "^1.6.4",
+    "react-select": "^3.0.4"
```
Comment (magul) [inline package.json]: Do we use that dependency?

File: src/components/Dropdown/Dropdown.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Dropdown from "react-dropdown";
+import styles from "./Dropdown.scss";
+
+const options = ["szkola1", "szkola2", "szkola3"];
+
+class Drop extends React.Component {
```
Comment (magul) [inline src/components/Dropdown/Dropdown.js]: There's no reason to save keystrokes here. We can name it `DropdownComponent` as `Drop` is pretty unclear.

File: src/components/Dropdown/Dropdown.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Dropdown from "react-dropdown";
+import styles from "./Dropdown.scss";
+
+const options = ["szkola1", "szkola2", "szkola3"];
```
Comment (magul) [inline src/components/Dropdown/Dropdown.js]: `options` for Dropdown should be passed using `constructor`, not as a constant array inside component module.

File: src/components/Dropdown/Dropdown.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+  render() {
+    const defaultOption = this.state.selected;
+    return (
+      <div className={styles.DropdownControl}>
+        <Dropdown
+          options={options}
+          onChange={this._onSelect}
+          value={defaultOption}
+          placeholder="Wybierz szkołę"
```
Comment (magul) [inline src/components/Dropdown/Dropdown.js]: passed to `constructor`, not hardcoded

File: src/components/Dropdown/Dropdown.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  render() {
+    const defaultOption = this.state.selected;
+    return (
+      <div className={styles.DropdownControl}>
+        <Dropdown
+          options={options}
+          onChange={this._onSelect}
+          value={defaultOption}
```
Comment (magul) [inline src/components/Dropdown/Dropdown.js]: How do `value={defaultOption}` and `placeholder` work together?

**Comment by magul** [comment]
+ you have merge conflicts to be resolved.

**Comment by Rafalkufel** [comment]
Because we are using `react-final-form` we don't need separate dropdown anymore.

**Pull request closed without merging**